pytest_twisted.py

```diff
@@ -167,61 +159,96 @@ def decorator(f):
     return fixture
 
 
+_mark_attribute_name = '_pytest_twisted_mark'
 async_fixture = _marked_async_fixture('async_fixture')
 async_yield_fixture = _marked_async_fixture('async_yield_fixture')
 
 
+def pytest_fixture_setup(fixturedef, request):
+    """Interface pytest to async setup for async and async yield fixtures."""
+    # TODO: what about inlineCallbacks fixtures?
+    maybe_mark = getattr(fixturedef.func, _mark_attribute_name, None)
```

Comment:
Maybe I'm not following the code exactly, but the `maybeDeferred` in `in_reactor` later on should handle that, I think?

Reply:
And this is why I need these todo's so I come back and get a clue...  `:|`  I was thinking there were `inlineCallbacks` fixtures and they ought to be handled here too.  There aren't.  \*smh\* and such.  Thanks.

Though, seems like they could be implemented (after 30 seconds of thought anyways).  Add a decorator to mark them and intercept them here for setup before pytest gets confused by them.